ui/perfherder/compare/CompareTableView.jsx

```diff
@@ -131,9 +131,7 @@ export default class CompareTableView extends React.Component {
     }
 
     if (hasSubtests) {
-      let subtestName = data[0].name.split(' ');
```

Comment:
This isn't correct. We're basically displaying a random subtest's full name, when instead we should display the suites fullname.
After all, we're listing all the subtests that suite has...